### PR TITLE
fix: sidebar css for displaying checkboxes no matter the screen size

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: src/js/pages/OffersList/components/SorterFilter.scss [hunk 1]
```diff
@@ -40,7 +40,7 @@ $menu-width: 85%;
 
   &.open {
     width: $menu-width;
-    padding: 24px $filter-menu-padding-x;
+    padding: 24px;
```
Comment (jrainville) [inline src/js/pages/OffersList/components/SorterFilter.scss]: Does this make the padding too big for small screens?

Comment (richard-ramos) [reply src/js/pages/OffersList/components/SorterFilter.scss]: In small screens it looked okay, however at bigger screen sizes this happens:
Screenshot

Comment (jrainville) [reply src/js/pages/OffersList/components/SorterFilter.scss]: This screen capture is before or after the fix?

Comment (richard-ramos) [reply src/js/pages/OffersList/components/SorterFilter.scss]: before the fix

Comment (jrainville) [reply src/js/pages/OffersList/components/SorterFilter.scss]: Ah ok, I'm curious now with the fix if on small screens the padding is too big

**Pull request merged**